tests/terratest/config/config.go

```diff
@@ -0,0 +1,98 @@
+package config
```

Comment:
I'm with Izaac, I think it would be better if you read the hardcoded values from a config file, rather than it be hardcoded in go. You can unmarshall form a json or yaml. That way it would give you more flexibility when certain values change in the config file.